scripts/backup

```diff
@@ -61,8 +63,6 @@ ynh_print_info --message="Backing up the MySQL database..."
 
 ynh_mysql_dump_db --database="$db_name" > db.sql
 
-(cd $install_dir && ynh_exec_as "$app" php$phpversion bin/console maintenance 0)
```

Comment:
This still needs to be done in case of a successful run.

Reply:
It's now part of `trap` hence will happen no matter if `backup` succeeds or fails.